webpack/components/OrganizationCheck/index.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Grid, Row, Col } from 'patternfly-react';
+import './index.scss';
+
+const OrganizationCheck = (props) => {
+  const { children, hide } = props;
+  if (hide) {
+    return (
+      <Grid bsClass="container-fluid">
+        <Row>
+          <Col sm={12}>
+            <div className="centered-message">
```

Comment:
Could you just use [`.text-center`](https://getbootstrap.com/docs/3.3/css/#type-alignment) here?